rudder-server/SOURCES/mvn-like

```diff
@@ -28,3 +28,10 @@ else
 fi
 curl -s -o "${outfile}" "${url}"
 
+# Set a minimum size of 1Mb
+minimumsize=1000000
+actualsize=$(wc -c <"${outfile}")
+if [ ${actualsize} -le ${minimumsize} ]; then
```

Comment:
why not use `stat -c %s `, `wc` does not seem evident to me

Reply:
wc -c is just counting the bytes, it's in the usual answers about checking size of a file.